nft_ingester/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "nft_ingester"
-version = "0.7.2"
+version = "0.1.0"
```

Comment:
are we sure about this?

Reply:
Yes, the first release will be 0.5. the current 0.7.2 is defined only here and is a leftover, most services has own version of 0.1.0.